public/pages/Rules/containers/Rules/Rules.tsx

```diff
@@ -119,7 +119,9 @@ export const Rules: React.FC<RulesProps> = (props) => {
           <EuiSpacer size={'m'} />
         </EuiFlexItem>
         <EuiFlexItem>
-          <RulesTable loading={loading} ruleItems={ruleItems} showRuleDetails={setFlyoutData} />
+          <EuiPanel>
```

Comment:
We don't need this panel. It will add an extra layer around the table as shown below:
![image](https://user-images.githubusercontent.com/114732919/215892671-ffb0dcd8-03ba-4211-ab71-453b99dd5116.png)

Reply:
It should not, I removed that panel from RulesTable.tsx, table should be a clean table, and add that wrapper where needed.

Reply:
Got it